internal/resource/config/idp/spconnection/idp_sp_connection_resource.go

```diff
@@ -270,13 +272,6 @@ var (
 		"require_signed_attribute_query": types.BoolType,
 		"require_encrypted_name_id":      types.BoolType,
 	}
-	attributeQueryAttrTypes = map[string]attr.Type{
```

Comment:
Why are only some of these moved to common? I assume because the remainder have encryptedValue somewhere nested underneath?

Reply:
correct, only some contain the encryptedValue so I kept the ones the data source uses in its file only